src/geometry.rs

```diff
@@ -76,6 +77,12 @@ pub enum Value {
     GeometryCollection(Vec<Geometry>),
 }
 
+impl fmt::Display for Value {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{:?}", self)
+    }
+}
```

Comment:
where does this get used?

Reply:
Required because the error definition that displays `geometry::Value` (`InvalidGeometryConversion`) needs a `fmt::Display` impl. I thought `Debug` would cover it, but alas…